### ISSUE TITLE
Add ARIA props

### ISSUE THREAD

**Issue by cmeeren** [Zaid-Ajaj/Feliz#41]
ARIA props should be added, e.g. `aria-labelledby`, `aria-describedby`, `aria-label`, etc.

The properties are listed [here](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties) (e.g. section 6.6).

I know basically nothing about these, just enough to know they are crucial to accessibility.

**Comment by Zaid-Ajaj** [comment]
Sounds like a good idea, PR please? 🙏

**Comment by cmeeren** [comment]
Sure, will try to have a look at it during the next few days 👍

**Comment by Zaid-Ajaj** [comment]
Awesome!

**Comment by cmeeren** [comment]
Which do you prefer?

```f#
prop.ariaDescribedBy
aria.describedBy
```

Also, given that the original names are e.g. `aria-describedby` which do you prefer?

```f#
ariaDescribedby
ariaDescribedBy
```

**Comment by Zaid-Ajaj** [comment]
prop.ariaDescribedBy all the way please (so without separate type and camelCased words)
